ldk/goja/src/clipboard/clipboard.test.ts

```diff
@@ -33,9 +34,17 @@ describe('Clipboard', () => {
   });
 
   describe('listen', () => {
+    it('setting clipboard olive helps configuration', () => {
```

Comment:
Minor point: I like to have test names read Clipboard -> Listen -> 'sets clipboard olive helps configuration'. What you have reads: Clipboard -> Listen -> 'setting clipboard olive helps configuration'.

Reply:
good point, fixed.